### PR TITLE
fix: links

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

File: ecma402/finished-proposals.md [hunk 1]
```diff
@@ -13,11 +13,10 @@ Finished proposals are proposals that have reached stage 4, and are included in
...
-| [`Intl.DisplayNames v2`][intl.displaynames-v2]                                               | Frank Tang                           | Frank Tang                           | December&nbsp;2021                                                | 2022              |
-
+| [`Intl.DateFormat.prototype.formatRange`][formatrange]                                       | Felipe Balbontín                     | Sathya Gunasekaran                   | [January 2021][formatrange-notes]                                                 | 2021              |
+| [`Intl.Segmenter`: Unicode Segmentation in JavaScript][intl-segmenter]                       | Daniel Ehrenberg, Richard Gibson     | Richard Gibson                       | [October 2021][intl-segmenter-notes]                                                 | 2022              |
+| [Extend TimeZoneName Option Proposal][extend-timezonename]                                   | Frank Tang                           | Frank Tang                           | December&nbsp;2021                                                | 2022              |
```
Comment (septs) [inline ecma402/finished-proposals.md:18]: unifi "frank tang" name?

see https://github.com/tc39/proposals/search?q=Frank

CC @FrankYFTang

File: ecma402/finished-proposals.md [hunk 2]
```diff
@@ -13,11 +13,10 @@ Finished proposals are proposals that have reached stage 4, and are included in
...
+| [`Intl.DisplayNames v2`][intl.displaynames-v2]                                               | Frank Tang                           | Frank Tang                           | [December 2021][intl.displaynames-v2-notes]                                                | 2022              |
```
Comment (ljharb) [inline ecma402/finished-proposals.md:19]: This still links to September 2019

**Review by ljharb — CHANGES_REQUESTED** [review]
The notes link targets need to be updated too

File: ecma402/finished-proposals.md (hunk 1 above)
Comment (septs) [reply ecma402/finished-proposals.md:18]: CC @ljharb

Comment (ljharb) [reply ecma402/finished-proposals.md:18]: yes, they should all be the same; it's up to Frank what's used tho.

Comment (septs) [reply ecma402/finished-proposals.md:18]: https://github.com/search?q=org%3Atc39+Frank+Tang+FrankYFTang&type=Code

it seems the "frank tang" is used more

**Comment by ljharb** [comment]
@septs it'd still be good to get this landed; any chance you'd be willing to update it?

**Comment by septs** [comment]
forked repo is deleted

**Comment by ljharb** [comment]
ah, that's unfortunate :-/

**Pull request closed without merging**